frontend/app/admin/advertisement/edit/[advertisementId]/page.tsx

```diff
@@ -0,0 +1,22 @@
+import { AdverrisementEditor } from '../../../../../components/admin/pages/AdvertisementEditor'
+import { AdvertisementType } from '../../../../../types/AdvertisementType'
+
+export default async function AdminEditAdvertisement({
+  params,
+}: {
+  params: { advertisementId: string }
+}) {
+  const res = await fetch(
+    `${process.env.NEXT_PUBLIC_BACKEND_URL}/admin/advertisement/${params.advertisementId}`,
+    {
+      cache: 'no-cache',
+    },
+  )
```

Comment:
ここに通信方式を指定してください。
```suggestion
  const res = await fetch(
    `${process.env.NEXT_PUBLIC_BACKEND_URL}/admin/advertisement/${params.advertisementId}`,
    {
    method: 'ここ'
      cache: 'no-cache',
    },
  )
```
他の実装参考にしてみてください。